t/25-bugs.t

```diff
@@ -40,4 +42,30 @@ ok($bug->refreshed, 'bug refreshed');
 ok($bug->bugid,     'bugid matched');
 is($c->bugtitle_for('poo#200', $bug), "Bug referenced: poo#200\nfoo bar < \" & ß", 'bug title not already escaped');
 
+subtest 'Unreferenced bugs cleanup job works' => sub {
+    $bug = $bugs->get_bug('poo#201');
+    $bug = $bugs->get_bug('poo#202');
```

Comment:
These assignments look useless. And do you even need the `get_bug` calls? If you actually want to test whether these bugs exist at this point, use e.g. `ok($bugs->get_bug('poo#201'), 'bug poo#201 not cleaned up so far')`.

Reply:
`get_bug` will create the bug if it doesn't exist - I removed the assignment, though.

Reply:
`ok($bugs->get_bug('poo#201'), 'bug poo#201 not cleaned up so far')` will always succeed.

Reply:
Then add a comment stating what these 2 lines do. Note that the method `get_bug` is terribly named. I wouldn't assume that it can create bugs from its name.